src/FsAutoComplete.Core/ParseAndCheckResults.fs

```diff
@@ -25,45 +25,54 @@ type ParseAndCheckResults
 
   let logger = LogProvider.getLoggerByName "ParseAndCheckResults"
 
-  member __.TryGetMethodOverrides (lines: LineStr[]) (pos: pos) = async {
-    // Find the number of `,` in the current signature
-    let commas, _, _ =
-      let lineCutoff = pos.Line - 6
-      let rec prevPos (line,col) =
-        match line, col with
-        | 1, 1
-        | _ when line < lineCutoff -> 1, 1
-        | _, 1 ->
-           let prevLine = lines.[line - 2]
-           if prevLine.Length = 0 then prevPos(line-1, 1)
-           else line - 1, prevLine.Length
-        | _    -> line, col - 1
-
-      let rec loop commas depth (line, col) =
-        if (line,col) <= (1,1) then (0, line, col) else
-        let ch = lines.[line - 1].[col - 1]
-        let commas = if depth = 0 && ch = ',' then commas + 1 else commas
-        if (ch = '(' || ch = '{' || ch = '[') && depth > 0 then loop commas (depth - 1) (prevPos (line,col))
-        elif ch = ')' || ch = '}' || ch = ']' then loop commas (depth + 1) (prevPos (line,col))
-        elif ch = '(' || ch = '<' then commas, line, col
-        else loop commas depth (prevPos (line,col))
-      match loop 0 0 (prevPos(pos.Line, pos.Column)) with
-      | _, 1, 1 -> 0, pos.Line, pos.Column
-      | newPos -> newPos
-
+  member x.TryGetMethodOverrides (lines: LineStr[]) (pos: Pos) = async {
     // Get the parameter locations
     let paramLocations = parseResults.FindNoteworthyParamInfoLocations pos
+
     match paramLocations with
     | None ->
-      return ResultOrString.Error "Could not find parameter locations"
+      // if there were no parameter locations, try just the first position
```

Comment:
this is the big new feature: if there's no noteworthy params, find the closest ident and see if it's a method group that we can suggest instead